modules/codegen-plugin/src/sbt-test/codegen-plugin/defaults/src/main/scala/Main.scala

```diff
@@ -24,4 +27,22 @@ object Main extends App {
       println("failed")
       sys.exit(1)
   }
+
+  def buildSchemaIndex() =
+    ModelLoader
+      .load(
+        specs = Set.empty,
```

Comment:
might wanna add some smithy file here too, to ensure its schemas are included in the loaded model

Reply:
with specs or not, this triggered the problem we observed initially

Reply:
yeah :)